ppr-ui/src/views/mhrSearch/ConfirmMHRSearch.vue

```diff
@@ -195,7 +196,11 @@ export default class ConfirmMHRSearch extends Vue {
 
   private get feeQuantity (): number {
     // Return selected quantity that is not a combination search
-    return this.getSelectedManufacturedHomes.filter(result => result.selected && !result.includeLienInfo).length
+    const uniqueRegNums = []
+    this.getSelectedManufacturedHomes.forEach(result => {
+      if (!uniqueRegNums.includes(result.mhrNumber)) uniqueRegNums.push(result.mhrNumber)
+    })
+    return uniqueRegNums.length
   }
```

Comment:
This function looks like its purpose has changed.
Is it still returning selected quantity that is NOT a combination search, ie `includeLienInfo`?

Reply:
refactored